backend/de.metas.adempiere.adempiere/base/src/main/java/de/metas/process/ProcessExecutionResult.java

```diff
@@ -110,13 +117,16 @@ public static ProcessExecutionResult newInstanceForADPInstanceId(final PInstance
 	 */
 	private boolean timeout = false;
 
-	/**
-	 * Log Info
-	 */
-	@Nullable
-	private transient List<ProcessInfoLog> logs;
 	private ShowProcessLogs showProcessLogsPolicy = ShowProcessLogs.Always;
 
+	private final ISysConfigBL sysConfigBL = Services.get(ISysConfigBL.class);
+	private final transient Debouncer<ProcessInfoLog> logsDebouncer = Debouncer.<ProcessInfoLog>builder()
+			.name("logsDebouncer")
```

Comment:
which logs? please make it clearer. e.g. use `ProcessExecutionResult.class.getName()+".Debouncer"`